api/v1beta1/clusterrelocation_types.go

```diff
@@ -69,6 +77,7 @@ type ClusterRelocationStatus struct {
 //+kubebuilder:resource:scope=Cluster
 
 // ClusterRelocation is the Schema for the clusterrelocations API
+// +operator-sdk:csv:customresourcedefinitions:resources={{Secret,v1alpha1,generated-api-secret},{Secret,v1,"generated-ingress-secret"}}
```

Comment:
```
{{Secret,v1alpha1,generated-api-secret},{Secret,v1,"generated-ingress-secret"}}
```
should be:
```
{{Secret,v1,"generated-api-secret"},{Secret,v1,"generated-ingress-secret"}}
```

It doesn't seem to matter if the names are quoted or not, but I think it should be consistent (either all quoted ,or all not quoted)